features/situation-feedback/elastic/src/main/java/org/opennms/features/situationfeedback/elastic/FeedbackDocument.java

```diff
@@ -128,7 +128,7 @@ public void setUser(String user) {
 
     public static FeedbackDocument from(AlarmFeedback feedback) {
         FeedbackDocument doc = new FeedbackDocument();
-        doc.setTimestamp(System.currentTimeMillis());
+        doc.setTimestamp(feedback.getTimestamp());
```

Comment:
Should we fallback to using `System.currentTimeMillis()` if the timestamp is `null`?

Reply:
Current behaviour is that you can't construct an `AlarmFeedback` with a null timestamp. If no timestamp is explicitly provided in the builder, it already defaults to using the time the builder was created.